src/actions/actions_custom.py

```diff
@@ -0,0 +1,30 @@
+from talon import Module, app, actions
+from ..csv_overrides import init_csv_and_watch_changes
+
+custom_action_defaults = {}
```

Comment:
Technically we could merge some things here that actually uses this kind of logic. Paste to for example actually used as the same logic as the custom action we have just implemented its internally.

Reply:
Hmm I'd be tempted not to do it for that one though as I still have half a mind to switch to our own impl at some point

But the point is a good one: I wonder which other ones we want to migrate here.  The only problem is that I think that would cause breakage because then we'd be removing them from the existing action csv, no?

Reply:
Exactly we would be moving them from one csv to another